modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc

```diff
@@ -1941,7 +5512,7 @@ NOTE: This input source does not support having multiple event notifications on
 |===
 
 [[custom-ocsp-responder]]
-=== Custom OCSP Responder
+=== Custom Ocsp Responder
```

Comment:
```suggestion
=== Custom OCSP Responder
```

Reply:
I think this is an actual thing in the API. Will follow up on these at a later date.